django_lightweight_queue/backends/redis.py

```diff
@@ -1,8 +1,13 @@
+import datetime
+
 import redis
 
 from .. import app_settings
 from ..job import Job
 from .base import BaseBackend
+from ..utils import block_for_time
+
+QueueName = str
```

Comment:
Alias for now; this will be replaced by an import once https://github.com/thread/django-lightweight-queue/pull/42